### ISSUE TITLE
Nickname authentication

### ISSUE THREAD

**Issue by k0nsl** [g0v/Logbot#33]
Consider adding nickname authentication with ordinary IRC server services such as `NickServ`.

**Comment by godfat** [comment]
Do you mean the bot should authenticate itself?

**Comment by k0nsl** [comment]
@godfat: Yep, exactly. Roughly my idea of it just means that the bot administrator can pre-register a nickname for the bot, then supply the password in the bot configuration and have the bot authenticate to `NickServ`.

**Comment by godfat** [comment]
@k0nsl Then this should be simple to implement. We could make it that providing a password, then the bot would try to authenticate itself, otherwise skip that step. Not sure if we could do this in Cinch already though? Maybe it's already there :P
